packages/material-ui/src/Chip/Chip.js

```diff
@@ -8,6 +8,7 @@ import useForkRef from '../utils/useForkRef';
 import unsupportedProp from '../utils/unsupportedProp';
 import capitalize from '../utils/capitalize';
 import ButtonBase from '../ButtonBase';
+import IconButton from '../IconButton';
```

Comment:
What do you think of not using the icon button component for this problem? Keeping it minimal?

Reply:
I think I agree here. Need to watch out why I went from ButtonBase to IconButton.